### PR TITLE
Release v0.26.1: bring "fix panic at start" commit (#2243) into stable

### PR DESCRIPTION
**Pull request by curquiza**
2 commits
- cherry pick `32843f30d973349122ec5a37469ee09e6002b6f3` (merged in #2243)
- change the version in the Cargo toml files (from v0.26.0 to v0.26.1)

### CODE REVIEW

**Comment by curquiza** [comment]
bors try

**Review by Kerollmops — APPROVED** [review]

**Comment by curquiza** [comment]
Let's wait for the @MarinPostma approval as well before merging!

**Comment by irevoire** [comment]
Maybe it would be better to remove the panic instead of bringing it into stable?

**Comment by curquiza** [comment]
Do you mean the title? :) Is it better now?

**Comment by MarinPostma** [comment]
Bors merge

**Pull request merged**
